Makefile

```diff
@@ -45,7 +45,7 @@ help: ## Display this help.
 
 .PHONY: run
 run: ## Run in development mode
-	go run cmd/aws-application-networking-k8s/main.go --debug -v 6
+	go run cmd/aws-application-networking-k8s/main.go --debug
```

Comment:
Why remove the `-v 6`? This removes logs that are still necessary for debugging until we fully migrate to zap logger